capsules/core/src/driver.rs

```diff
@@ -26,6 +26,8 @@ pub enum NUM {
 
     // Kernel
     Ipc                   = 0x10000,
+    ProcLoader            = 0x10002,
+    MMU                   = 0x10003,
```

Comment:
?

Reply:
ProcLoader slipped in.

MMU should read MPU and will make sense when we manage to commit the CHERI MPU (which was a little hard to dis-entangle). We can remove it for now.

The CHERI "MPU" can be directly given commands from userspace to ask it to perform fine-grained access control for the user heap.